cmd/relay.go

```diff
@@ -1,104 +0,0 @@
-package cmd
```

Comment:
Let's add an issue to the last milestone to implement the relay command with a reference to this PR where we temporarily remove it.